public/components/home/home.controller.js

```diff
@@ -10,12 +10,16 @@ angular.module('textAnalysis')
             };
 
             $scope.showSideNav = function() {
+                console.log($scope.userInput);
+                console.log($scope.submitHandle.userInput);
```

Comment:
I'll be merging this code, but, for a future PR, remove these console logs from production code